distribution/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:6.10
+RUN yum install --assumeyes gcc
```

Comment:
there's a funny bit in the latest season of arrested development that this reminds me of

![presume teeth](https://www.reddit.com/r/arresteddevelopment/comments/b9dob3/uncheck_presume_teeth_im_dying_lmao/)